src/NuGet.Core/NuGet.Frameworks/NuGetFramework.cs

```diff
@@ -166,7 +166,7 @@ public virtual string GetShortFolderName(IFrameworkNameProvider mappings)
                     throw new FrameworkException(string.Format(
                         CultureInfo.CurrentCulture,
                         Strings.InvalidFrameworkIdentifier,
-                        shortFramework));
+                        framework.Framework));
```

Comment:
Please add a test to `test/NuGet.Core.Tests/NuGet.Frameworks.Test/NuGetFrameworkTests.cs` to ensure the framework name is in the message